controllers/controller.js

```diff
@@ -42,4 +42,16 @@ exports.postNewComment = (req,res, next)=>{
     insertNewComment(article_id, commentData).then((newCommentsList)=>{
         res.status(201).send({comments: newCommentsList})
     }).catch(next)
+}
+
+exports.deleteComment = (req, res, next)=>{
+    const {comment_id} = req.params
+    const commentPromises = [deleteCommentById(comment_id)]
+    if(comment_id){
+        commentPromises.push(checkExists("comments", "comment_id", comment_id))
+    }
+    Promise.all(commentPromises).then((resolvedPromises)=>{
+        const deletedComment = resolvedPromises[0]
+        res.status(204).send({deletedComment})
```

Comment:
We shouldn't send anything back from this endpoint here. You can just invoke .send() or even do this -  res.sendStatus(204)